src/shared/constants.h

```diff
@@ -53,6 +53,8 @@ const double METERS_PER_CENTIMETER = 1.0 / 100.0;
 const double CENTIRADIANS_PER_RADIAN = 100.0;
 const double RADIANS_PER_CENTIRADIAN = 1.0 / 100.0;
 
+const double NANOSECONDS_PER_MILLISECOND  = 1000000.0;
+const double NANOSECONDS_PER_SECOND       = 1000000000.0;
```

Comment:
At this many zeros, could we use `1e9` instead? I suppose we should change the other constants here to be consistent as well (but this last bit is technically out-of-scope, so optional).

Reply:
Yeah this whole file ought to be changed all at once